communication/resources/base.css

```diff
@@ -1,9 +1,16 @@
+:root {
+    --neutral-background: #1c1913 !important;
+    --neutral-text: #ffffc7 !important;
+    --selection-background: #5b513f !important;
+    --selection-text: #ffffd8 !important;
+}
+
 html, body {
     height: 100vh;
 }
 
 body {
-    font-family: 'Montserrat';
+    font-family: 'Montserrat', serif;
```

Comment:
Forgot to mention in the commit, but I added a generic font-family fallback here.

Reply:
What for though? The font is shipped with scribble.rs

Reply:
Just general good practice AFAIK.

Also my IDE kept throwing errors to fix it.